.github/workflows/ci-pipeline.yml

```diff
@@ -41,20 +41,12 @@ jobs:
           target: production
           push: true
           tags: ${{ secrets.DOCKERHUB_USERNAME }}/todoapp:latest,${{ secrets.DOCKERHUB_USERNAME }}/todoapp:${{ github.sha }}
-  deploy-to-heroku:
+  deploy-to-azure:
     needs: [build-docker-image]
-    name: Deploy to Heroku
+    name: Deploy to Azure
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Release to Heroku
-        uses: gonuit/heroku-docker-deploy@v1.3.3
-        with:
-          email: ${{ secrets.HEROKU_EMAIL }}
-          heroku_api_key: ${{ secrets.HEROKU_TOKEN }}
-          heroku_app_name: ${{ secrets.HEROKU_APP_NAME }}
-          dockerfile_directory: ./
-          dockerfile_name: Dockerfile
-          docker_options: "--target production"
-          process_type: web
\ No newline at end of file
+      - name: Release to Azure
+        run: curl -dH -X POST '${{ secrets.AZURE_WEBHOOK_URL }}'
```

Comment:
Nice, one small improvement I find is to add the `--fail` flag to your curl command; without it this step will pass as long as it successfully gets a response, _even if that response is a failure response like a 404_!